### ISSUE TITLE
Edge default datetime prop mismatch

### ISSUE THREAD

**Issue by kikimo** [vesoft-inc/nebula#3045]
**Please check the FAQ documentation before raising an issue**

Please check the [FAQ](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-EN/1.overview/2.quick-start/2.FAQ.md) documentation and old issues before raising an issue in case someone has asked the same question that you are asking.

**Describe the bug (__must be provided__)**

Edge default datetime prop mismatch.

**Your Environments (__must be provided__)**

* OS: `uname -a`
* Compliler: `g++ --version` or `clang++ --version`
* CPU: `lscpu`
* Commit id 030e5a93abf53e8f60778af160bb222e20351cf5 (it's a branch containing all v2.6 commits, and @liuyu85cn is fixing TOSS problem there, but the mismatch has nothing to do with TOSS).

**How To Reproduce(__must be provided__)**

1. start a nebula cluster
2. create edge known(ts datetime default datetime(), idx string);
3. start a client and insert 4096 edges without specifying ts field

check equality of kdata field of in/out edge, you will find a lot of edge prop mismatch, the discrepancy  generally come form Microsec field of ts value, eg.

```txt
...
prop mismatch in edge 123->57, forward vs backward, idx: 333->15801 == 333->15801, ts: DateTime({Year:2021 Month:10 Day:13 Hour:1 Minute:56 Sec:21 Microsec:710000}) != DateTime({Year:2021 Month:10 Day:13 Hour:1 Minute:56 Sec:21 Microsec:709000})
prop mismatch in edge 105->68, forward vs backward, idx: 615->13508 == 615->13508, ts: DateTime({Year:2021 Month:10 Day:13 Hour:1 Minute:55 Sec:18 Microsec:680000}) != DateTime({Year:2021 Month:10 Day:13 Hour:1 Minute:55 Sec:18 Microsec:675000})
prop mismatch in edge 22->81, forward vs backward, idx: 547->2897 == 547->2897, ts: DateTime({Year:2021 Month:10 Day:13 Hour:1 Minute:53 Sec:4 Microsec:963000}) != DateTime({Year:2021 Month:10 Day:13 Hour:1 Minute:53 Sec:4 Microsec:920000})
...
```

**Expected behavior**

We expect the value of ts field to be equal in in/out edge of an edge.

**Additional context**

Provide logs and configs, or any other context to trace the problem.

**Comment by liuyu85cn** [comment]
This happened while instead of use the default GraphStorageClient, kiki called storage server API directly. 
(for performance testing)
The AddEdgesRequest passed to storage has 1 propName and 2 vals. 

Normally, when we used storageClient, 
it will either
1. leave propName empty
2. make sure propName has the same size of prop vals. 

This won't happen in user case. move to 3.0 to dig more.

**Comment by critical27** [comment]
Is this issue fixed?

**Comment by kikimo** [comment]
> Is this issue fixed?

not yet

**Comment by critical27** [comment]
Fixed in #3880
